types/httpstatus.pp

```diff
@@ -1,2 +1,2 @@
 # @summary Validate a HTTP status code
```

Comment:
I think you can deprecate this type. The `@see` may not be needed if `@deprecated` creates a link, but I don't think it does.
```suggestion
# @summary Validate a HTTP status code
# @deprecated Use Stdlib::Http::Status
# @see Stdlib::Http::Status
```

Please check by regenerating `REFERENCE.md`.

Reply:
i have added both but i don't think deprecated dose anything yet https://github.com/puppetlabs/puppet-strings/issues/339